crates/bevy_input/src/gamepad/button.rs

```diff
@@ -0,0 +1,64 @@
+use crate::gamepad::{Gamepad, GamepadButtonType};
+
+/// A button of a [`Gamepad`].
+///
+/// ## Usage
+///
+/// It is used as the generic `T` value of an [`Input`](crate::Input) and [`Axis`](crate::Axis)
+/// to create `Bevy` resources. These resources store the data of the buttons and axes of a gamepad
+/// and can be accessed inside of a system.
+///
+/// ## Access
+///
+/// To access the resources use one of the following:
```

Comment:
Similar comment as above, although it's useful to explain how to access both the input and access.